packages/blade/src/components/BottomSheet/_decisions/decisions.md

```diff
@@ -0,0 +1,225 @@
+# Bottom Sheet
+
+Bottom sheet is a container that presents additional information to a user in an overlay.
+
+This document outlines the API of `BottomSheet` component.
+
+<img src="./bottom-sheet-thumbnail.png" width="380" alt="" />
+
+## Design
+
+- [Figma - BottomSheet](https://www.figma.com/file/jubmQL9Z8V7881ayUD95ps/Blade---Payment-Light?node-id=26477%3A578228&t=1RMqjs99P33Udj0C-0)
+
+## API
+
+Sample usage:
+
+```jsx
+import { BottomSheet } from '@razorpay/blade';
+
+<BottomSheet open={boolean} snapPoints={[]}>
+  <BottomSheetHeader>
+    <BottomSheetHeaderLeading title="Payments Links" prefix={BladeIcon} />
+    <BottomSheetHeaderTrailing visual={Link | Action | Icon} />
+  </BottomSheetHeader>
+  <BottomSheetBody>Body Content</BottomSheetBody>
+  <BottomSheet>
+    <BottomSheetLeading title="Footer Title" prefix={icon} />
+    <BottomSheetTrailing
+      actions={{
+        primaryAction: { text: 'Know more', onClick: () => {} },
+        secondaryAction: { text: 'Read Docs', onClick: () => {} },
+      }}
+    />
+  </BottomSheet>
+</BottomSheet>;
+```
+
+**Design Constraints**
+
+- BottomSheetHeader:
+
+  - Leading:
+    - Title
+    - Prefix
+  - Trailing:
+    - empty
+    - icon
+    - link
+    - action
+
+- BottomSheetFooter:
+
+  - Leading
+    - title
+    - prefix
+  - Trailing
+    - Primary action button
+    - Secondary action button
+
+- The BottomSheetFooter will behave similar to Card's footer.
+- The Header, Footer can be individually omitted
+
+### `BottomSheet`
+
+We'll expose an `BottomSheeet` component with the following API:
+
+| Prop            | Type        | Default                 | Description                                                                                     | Required |
+| --------------- | ----------- | ----------------------- | ----------------------------------------------------------------------------------------------- | -------- |
+| open            | `boolean`   | `false`                 | toggles bottom sheet state content                                                              |          |
+| snapPoints      | `string[]`  | `['25%', '50%', '85%']` | snappoints in which the bottom sheeet will rest on                                              |          |
+| onDismiss       | `Callback`  | `undefined`             | called when the bottom sheet is closed, either by user state, hitting `esc` or tapping backdrop |          |
+| initialFocusRef | `React.Ref` | `undefined`             | ref element you want to get keyboard focus when opening the sheet                               |          |
+
```

Comment:
Is there any controls for the close button?

Reply:
No not sure it's needed because the "close" button will only have 1 action "closing the sheet", so instead if users want to react to or know the state of the sheet from outside they can use the onDismiss prop (which will get triggered when we click the close button)

Reply:
So, will we always show the close button?

Reply:
Yes if the bottomsheet renders the header, it will always have the close button.